docs/cloud-adoption/governance/governance-multiple-teams.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: "Governance design guide: new development in Azure for multiple teams"
+title: "Enterprise Cloud Adoption: Governance design in Azure for multiple teams"
 description: Guidance for configuring Azure governance controls for multiple teams, multiple workloads, and multiple environments
-author: petertay
+author: petertaylor9999
 ---
 
-# Governance design guide: new development in Azure for multiple teams
+# Enterprise Cloud Adoption: Governance design for multiple teams
 
-The goal of this guidance is to help you learn the process for designing a resource governance model in Azure.  We'll look at a set of hypothetical governance requirements, then go through several example implementations that satisfy those requirements. 
+The goal of this guidance is to help you learn the process for designing a resource governance model in Azure to support multiple teams, multiple workloads, and multiple environments.  We'll look at a set of hypothetical governance requirements, then go through several example implementations that satisfy those requirements. 
 
 Our requirements are:
 * Identity management for multiple teams with different resource access requirements in Azure. The identity management system stores the identity of the following users:
```

Comment:
Below,

> As you learned earlier, an environment is a logical grouping of resources

Looking through the earlier topics I don't see a definition of "environment" - maybe just define it here (which you do anyway)